src/main/java/uk/gov/hmcts/reform/professionalapi/domain/entities/ProfessionalUser.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.professionalapi.domain.entities;
+
+import java.util.UUID;
+import javax.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity(name = "professional_user")
+@Getter
+@NoArgsConstructor
+public class ProfessionalUser {
+
+    @Id
+    private String id;
```

Comment:
Same as the Organistion entity, this would be much better using UUIDs and not strings for the ID, probably via a superclass